python/lsst/ts/Dome/mock_controller.py

```diff
@@ -131,132 +207,260 @@ async def cmd_loop(self, reader, writer):
+        )
+
+    async def crawlEl(self, **kwargs):
+        """Crawl the light and wind screen.
+
+        Parameters
+        ----------
+        kwargs: `dict`
+            A dictionary with arguments to the function call. It should contain the key "dirMotion" with a
+            string value (UP or DOWN) and the key "elRate" with a float value.
+        """
+        self.log.info(f"Received command 'crawlEl' with arguments {kwargs}")
+
+        # No conversion from radians to degrees needed since both the commands and the mock az controller
+        # use degrees.
+        # TODO Make sure that radians are used because that is what the real LLCs will use as well. DM-24789
+        await self.lwscs.crawlEl(
+            direction=kwargs["dirMotion"], velocity=float(kwargs["elRate"])
+        )
+
+    async def setLouver(self, **kwargs):
+        """Set the position of a louver.
+
+        Parameters
+        ----------
+        kwargs: `dict`
+            A dictionary with arguments to the function call. It should contain the key "id" with an int
+            value and the key "position" with a float value.
+        """
+        self.log.info(f"Received command 'setLouver' with arguments {kwargs}")
+
+        # No conversion from radians to degrees needed since both the commands and the mock az controller
+        # use degrees.
+        # TODO Make sure that radians are used because that is what the real LLCs will use as well. DM-24789
+        await self.lcs.setLouver(
+            louver_id=int(kwargs["id"]), position=int(kwargs["position"])
+        )
+
+    async def closeLouvers(self):
+        """Close all louvers.
+        """
+        self.log.info(f"Received command 'closeLouvers'")
+        await self.lcs.closeLouvers()
+
+    async def stopLouvers(self):
+        """Stop the motion of all louvers.
+        """
+        self.log.info(f"Received command 'stopLouvers'")
+        await self.lcs.stopLouvers()
+
+    async def openShutter(self):
+        """Open the shutter.
+        """
+        self.log.info(f"Received command 'openShutter'")
+        await self.apscs.openShutter()
+
+    async def closeShutter(self):
+        """Close the shutter.
+        """
+        self.log.info(f"Received command 'closeShutter'")
+        await self.apscs.closeShutter()
+
+    async def stopShutter(self):
+        """Stop the motion of the shutter.
+        """
+        self.log.info(f"Received command 'stopShutter'")
+        await self.apscs.stopShutter()
 
     async def config(self, **kwargs):
+        """Configure the lower level components.
+
+        Parameters
+        ----------
+        kwargs: `dict`
+            A dictionary with arguments to the function call. It should contain keys for all lower level
+            components to be configured with values that are dicts with keys for all the parameters that
+            need to be configured. The structure is
+            "AMCS":
```

Comment:
Please consider a preformatted block, because otherwise you will lose all formatting in the rendered sphinx (I thought you had made this change, but maybe not everywhere?):
```
"The structure is::

    "AMCS":
        "jmax"
...

It is assumed...
```

Reply:
Thanks for catching that one!